src/Parser/Rogue/Subtlety/Modules/Talents/DarkShadowEvis.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import Enemies from 'Parser/Core/Modules/Enemies';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import ShadowDance from './../Features/ShadowDance';
+import DarkShadow from './DarkShadow';
+
+
+class DarkShadowEvis extends DarkShadow {
+  static dependencies = {
+    enemies: Enemies,
+    combatants: Combatants,
+    shadowDance: ShadowDance,
+  };
+
+  statistic() {    
+    const danceEvis = this.shadowDance.totalEviscerateDamageInShadowDance / this.shadowDance.totalShadowDanceCast;
```

Comment:
`totalEviscerateDamageInShadowDance` is a pretty confusing name for what's actually 'total evis casts in shadow dance'. I understand you increment it on damage events, but still.